lib/UseFilePlugin.js

```diff
@@ -1,23 +1,29 @@
+"use strict";
 /*
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
-function UseFilePlugin(source, filename, target) {
-	this.source = source;
-	this.filename = filename;
-	this.target = target;
-}
-module.exports = UseFilePlugin;
+const assign = require("object-assign");
```

Comment:
Sure, for now just remove from this file for this PR, and then do the rest in a separate PR.